src/main/cpp/windows/jssc.cpp

```diff
@@ -136,7 +136,7 @@ JNIEXPORT jboolean JNICALL Java_jssc_SerialNativeInterface_setParams
 
         if(SetCommState(hComm, dcb)){
 
-        	//since 2.1.0 -> previously setted timeouts by another application should be cleared
+        	//since 2.1.0 -> previously set timeouts by another application should be cleared
```

Comment:
Can you fix the grammar all the way :)
```suggestion
        	//since 2.1.0 -> timeouts set previously by another application should be cleared
```